packages/editor/src/components/properties/NameInputGroup.tsx

```diff
@@ -54,14 +54,14 @@ const styledNameInputGroupStyle = {
  * @type {class component}
  */
 export const NameInputGroup: EditorComponentType = (props) => {
-  const nameComponent = useComponent(props.entity, NameComponent)
-
+  const nameComponent = useOptionalComponent(props.entity, NameComponent)
```

Comment:
Can we figure out instead why the name input group is being rendered?

Reply:
It's being rendered in the properties panel?

Reply:
Yeah, but I see your comment in the issue so this is fine